apps/website/src/components/modules/UserOnboardingProcess/UserOnboarding/UserOnboarding.js

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from 'react'
-import FormField from '../../../../../../packages/UI/src/components/organisms/FormField'
-import OpenResponse from '../../../../../../packages/UI/src/components/organisms/OpenResponse'
+import FormField from '../../../../../../../packages/UI/src/components/organisms/FormField'
+import OpenResponse from '../../../../../../../packages/UI/src/components/organisms/OpenResponse'
```

Comment:
use `@develaunchers/components/components/organisms/OpenResponse`

Reply:
I am getting this error
![2023-04-02 (1)](https://user-images.githubusercontent.com/110208240/229342832-2fe4b98d-c7fc-47ed-8988-fa634b61b597.png)
This is when I am using @develaunchers/components/components/organisms/OpenResponse

Reply:
I have try with @develaunchers/components/components/organisms/OpenResponse and @develaunchers/components/organisms/OpenResponse